charts/crd/templates/crd.yaml

```diff
@@ -3,7 +3,7 @@
 {{- if (semverCompare ">=1.19-0" (substr 1 -1 .Capabilities.KubeVersion.GitVersion)) }}
 apiVersion: apiextensions.k8s.io/v1
 {{- else }}
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
There is already a switch that checks the k8s version and adjusts the apiVersion accordingly.
AKS >=1.19.0 -> apiextensions.k8s.io/v1
AKS <1.19.0 -> apiVersion: apiextensions.k8s.io/v1beta1